### PR TITLE
just for explaining

### PR DESCRIPTION
**Pull request by lesia-grapeniuk**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,104 @@
...
+  height: 4px;
+  border-radius: 2px;
+  background-color: #00acdc;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  left: 50px;
```
Comment (Viktor-Kost) [inline src/style.css:97]: Remove the 'left' property since it has no effect on a block-level element.

File: src/style.css [hunk 2]
```diff
@@ -1 +1,104 @@
 /* Styles go here */
+body {
+  background: #e5e5e5;
+  margin: 0;
+}
+
+@media (max-width: 1024px) {
```
Comment (Viktor-Kost) [inline src/style.css:7]: No needs to use media queries for this task. You can just apply max-width to your header and place it at the center

File: src/style.css [hunk 3]
```diff
@@ -1 +1,104 @@
...
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+    padding: 0 50px;
+  }
+}
+
+@media (max-width: 1200px) {
```
Comment (Viktor-Kost) [inline src/style.css:20]: Replace max-width with min-width to fit the layout on 1024px and 1200px.

File: src/style.css [hunk 4]
```diff
@@ -1 +1,104 @@
...
+  border-radius: 2px;
+  background-color: #00acdc;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
```
Comment (Viktor-Kost) [inline src/style.css:98]: Remove the 'top' property since it has no effect on a block-level element.

File: src/index.html [hunk 5]
```diff
@@ -9,6 +9,44 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo" class="logo-image">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <a
```
Comment (Viktor-Kost) [inline src/index.html:18]: Use the 'li' tag for the list items inside the 'ul' tag
Example: 
```
<ul>
  <li>
    <a></a>
  </li>
  ....
  <li>
    <a></a>
  </li>
</ul>
```

File: src/index.html [hunk 6]
```diff
@@ -9,6 +9,44 @@
...
+            href="#Apple"
+            class="nav__link is-active"
+          >Apple
+          </a>
+          <a
+            href="#Samsung" class="nav__link"
+          >Samsung
+          </a>
```
Comment (Viktor-Kost) [inline src/index.html:26]: check code styling [recommendations](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
          <a href="#" class="nav__link">
            Samsung
          </a>
```

File: src/style.css [hunk 7]
```diff
@@ -1 +1,104 @@
...
+  display: block;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+}
+
+ul {
+  padding-inline-start: 40px;
+  justify-content: space-between;
+}
```
Comment (Viktor-Kost) [inline src/style.css:43]: style class selectors instead of tag ones

File: readme.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://lesia-grapeniuk.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://lesia-grapeniuk.github.io/layout_moyo-header/report/html_report/)
```
Comment (Viktor-Kost) [inline readme.md:4]: please add these links to Pull Request description

File: src/style.css [hunk 9]
```diff
@@ -1 +1,104 @@
...
+  text-decoration: none;
+  color: #00acdc;
+  border-radius: 8px;
+}
+
+.nav__link.is-active {
+  position: relative;
+  align-items: center;
+  color: #00acdc;
+  margin-left: 0;
```
Comment (Viktor-Kost) [inline src/style.css:79]: redundant
```suggestion
  position: relative;
  color: #00acdc;
```

File: src/style.css [hunk 10]
```diff
@@ -1 +1,104 @@
...
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 14px;
+  margin-left: 20px;
```
Comment (Viktor-Kost) [inline src/style.css:66]: 1) move margin-left: 20px; to `.nav__item` (after you add `<li>` to your html)
2) use `:not:(:first-child)` to add margin to all of your items besides first one

File: src/style.css [hunk 11]
```diff
@@ -1 +1,104 @@
...
+    padding: 0 50px;
+  }
+}
+
+.nav {
+  display: block;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+}
```
Comment (Viktor-Kost) [inline src/style.css:38]: redundant
```suggestion
```

File: src/style.css [hunk 12]
```diff
@@ -1 +1,104 @@
...
+  margin: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-content: space-between;
+  align-items: center;
+  height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:57]: it's better not hardcode height for elements with content inside. Use padding

File: src/style.css [hunk 13]
```diff
@@ -1 +1,104 @@
...
+
+@media (max-width: 1200px) {
+  .header {
+    background-color: #fff;
+    position: relative;
+    display: flex;
+    margin: 0;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
```
Comment (Viktor-Kost) [inline src/style.css:28]: redundant, since it's default value for display: flex
```suggestion
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 14]
```diff
@@ -1 +1,104 @@
...
+.nav__link:hover {
+  text-decoration: none;
+  color: #00acdc;
+  border-radius: 8px;
+}
+
+.nav__link.is-active {
+  position: relative;
+  align-items: center;
+  color: #00acdc;
```
[the remark this answers is not in the document]
Comment (lesia-grapeniuk) [reply src/style.css:76]: @Viktor-Kost it`s need for first active link. And on the figma and page in the BackstopJS Report we have also link "Apple"  with  active blue color.

File: src/index.html (hunk 5 above)
Comment (Viktor-Kost) [reply src/index.html:18]: isn't fixed

File: src/index.html (hunk 6 above)
Comment (Viktor-Kost) [reply src/index.html:26]: isn't fixed

File: src/style.css (hunk 7 above)
Comment (Viktor-Kost) [reply src/style.css:41]: isn't fixed

File: src/style.css [hunk 15]
```diff
@@ -1 +1,104 @@
...
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-content: space-between;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
```
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/style.css:56]: isn't fixed

File: readme.md (hunk 8 above)
Comment (Viktor-Kost) [reply readme.md:4]: isn't fixed

File: src/style.css (hunk 9 above)
Comment (Viktor-Kost) [reply src/style.css:77]: isn't fixed

File: src/style.css (hunk 10 above)
Comment (Viktor-Kost) [reply src/style.css:64]: isn't fixed

File: src/style.css (hunk 11 above)
Comment (Viktor-Kost) [reply src/style.css:36]: isn't fixed

File: src/style.css (hunk 14 above)
[the remark this answers is not in the document]
Comment (Viktor-Kost) [reply src/style.css:76]: yes, you're right. But I meant that you need to create variables for colors you use. It's a common practice. [Here](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) you can read how to do it

File: src/style.css (hunk 12 above)
Comment (Viktor-Kost) [reply src/style.css:55]: isn't fixed

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Hello!)
Pleas check all unresolved conversation to apply fixes. 
If you need any help - feel free to ask at fe_chat

**Pull request closed without merging**